core/src/main/java/org/apache/commons/vfs2/provider/AbstractFileSystem.java

```diff
@@ -317,42 +317,46 @@ public FileObject resolveFile(final FileName name) throws FileSystemException
         return resolveFile(name, true);
     }
 
-    private synchronized FileObject resolveFile(final FileName name, final boolean useCache) throws FileSystemException
```

Comment:
I've heard about http://vmlens.com/ , perhaps that will allow creating a reproducible test case.